bcipy/signal/model/offline_analysis.py

```diff
@@ -97,20 +98,20 @@ def offline_analysis(data_folder: str = None,
                                 channel_map=channel_map,
                                 trial_length=trial_length)
 
-    model, auc = train_pca_rda_kde_model(x, y, k_folds=k_folds)
+    model = PcaRdaKdeModel(k_folds=k_folds)
+    model.fit(x, y)
+    model_performance = model.evaluate(x, y)
 
     log.info('Saving offline analysis plots!')
 
-    # After obtaining the model get the transformed data for plotting purposes
-    model.transform(x)
     fig_handles = generate_offline_analysis_screen(
         x, y, model=model, folder=data_folder,
         down_sample_rate=downsample_rate,
         fs=fs, save_figure=True, show_figure=False,
         channel_names=analysis_channel_names_by_pos(channels, channel_map))
 
     log.info('Saving the model!')
-    with open(data_folder + f'/model_{auc}.pkl', 'wb') as output:
+    with open(data_folder + f'/model_{model_performance.auc}.pkl', 'wb') as output:
```

Comment:
Should we round this to a certain number of decimal places? It seems like we only use the first few and these file names get to be pretty unwieldy. @sci-tab , thoughts?

Reply:
Sounds like a good idea for a quick follow-up ticket

Reply:
https://www.pivotaltracker.com/story/show/178045471